docs/getting_started/installation.md

```diff
@@ -1,13 +1,42 @@
 # Installation
```

Comment:
I don't now whether we actually need any proper "Installation" if you work with flowcean, at the moment, using the hatch shell inside the cloned repository might be the best variant. As soon as we publish a proper release, we should also be pip-installable